### PR TITLE
Ports Pepperspray rework

### PR DESCRIPTION
**Pull request by VictorVow**
https://github.com/tgstation/tgstation/pull/46342

## About The Pull Request

Volume increased to 50 for pepper spray, added knockdown of decreasing degree for decreasing importance pepper spray entrance organs, added confuse, blind, blur, slow down. Adds a new clothing flag for head and mouth, PEPPERPROOF, that will decide if you can be pepper-sprayed while wearing those. Decreases scope of pepperspray protection by a lot.
## Why It's Good For The Game
 
Pepper spray is not even worth using right now, and I routinely see sec officers throwing away their pepper spray for e-bolas, handcuffs or flashbangs. This PR will make pepper spray a viable option in the sec arsenal, but does account for the range and spam ability of pepper spray while still making it competitive. As we are moving away from hardstuns, I see it fit to replace the hard stun with some bad effects, slowdown and knockdown. Helmets, Hardsuits, Gas masks, and other things like that are pepperproof
## Changelog

:cl:
balance: increased volume of pepper sprays to 50
tweak: adds knockdown of 3 second for no protection, 3 for no mouth protection
tweak: Changes bluring and blindness of eyes
balance: Replaces Paralysis with blind
balance: vastly decreased the scope of pepperspray protection.
 add: New variable for pepperspray protection, PEPPERPROOF, decides if you can be pepperproof with certain equipment
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
image

**Comment by VictorVow** [comment]
Whoops added something that shouldn't be there.

File: code/modules/clothing/glasses/hud.dm [hunk 1]
```diff
@@ -4,6 +4,8 @@
 	flags_1 = null //doesn't protect eyes because it's a monocle, duh
 	var/hud_type = null
 	var/alt_hud_type = null
+	///Used for topic calls. Just because you have a HUD display doesn't mean you should be able to interact with stuff.
+	var/hud_trait = null
```
Comment (ike709) [inline code/modules/clothing/glasses/hud.dm]: Unrelated change?

**Review by ike709 — CHANGES_REQUESTED** [review]
Address the comment.

File: code/modules/clothing/glasses/hud.dm (hunk 1 above)
Comment (VictorVow) [reply code/modules/clothing/glasses/hud.dm]: Ah yes, I'll remove.

**Comment by MarkSuckerberg** [comment]
Have you tested it?

**Review by Crossedfall — APPROVED** [review]
Seems to work within the test merge

**Pull request merged**